lib/database.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Database
+  SEED = './spec/fixtures/seed.yaml'
```

Comment:
fixtures file should be used only in tests
create a separate `database.yml` file for non-test environment